src/core_ocean/analysis_members/Registry_mixed_layer_depths.xml

```diff
@@ -77,7 +77,16 @@
 			 description="mixed layer depth based on gradient of density"
 		/>
 	</var_struct>
-	<streams>
+<var_struct name="mixedLayerDepthAMScratch" time_levs="1">
+▸\▸\<var name="pressureAdjustedForLandIceScratch"
```

Comment:
something weird happened here with the "tabs".  I think this is causing errors in the run, but I can't be sure.